recipes/biorbd-casadi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set libname = "biorbd" %}
+{% set name = "biorbd-casadi" %}
+{% set version = "1.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/{{ libname }}/archive/Release_{{ version }}.tar.gz
+  sha256: 7682fa0c34d6c9a4415c8293d12089845390516845334dbb5c4fcaedb60872ef
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36 or py>37]
+  rpaths:
+    - lib/
+    - lib/{{ libname }}/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - swig
+  host:
+    - casadi
+    - rbdl-casadi
+  run:
+    - casadi
+    - rbdl-casadi
+
+test:
+  commands:
+    - test -f $PREFIX/lib/biorbd/libbiorbd$SHLIB_EXT  # [unix]
+    - if not exist %PREFIX%\\Lib\\biorbd*.lib exit 1  # [win]
+  imports:
+    - biorbd
+
+about:
+  home: http://github.com/pyomeca/biorbd
+  license: MIT
+  license_family: MIT
```

Comment:
This isn't needed any more:
```suggestion
```
(it's added automatically when using SPDX identifiers)